project/attosoc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "srcs": [
+        "src/attosoc/attosoc.v",
+        "src/attosoc/attosoc_top.v"
+    ],
+    "data": [
+        "src/attosoc/firmware.hex"
+    ],
+    "top": "top",
+    "name": "attosoc",
+    "toolchains": {
+        "vivado": {
+            "arty": ["arty.xdc"]
+        },
+        "nextpnr": {
+            "arty": ["arty.xdc"]
+        }
```

Comment:
Should we have an issue to try adding vpr here once IO LOC support for symbiflow-arch-defs is added (e.g. https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1519 )

Reply:
I believe we can temporarily use PCF and convert it to XDC once the support is ready.

On a side note, with https://github.com/SymbiFlow/fpga-tool-perf/pull/136, nextpnr toolchain moved to be `nextpnr-xilinx` as I have added ice40 support (which resulted in the need renaming nextpnr toolchains).